README.md

```diff
@@ -1,4 +1,4 @@
-# Azure Resource Manager Tools
+# Azure Resource Manager Tools [![Build Status](https://travis-ci.org/Microsoft/vscode-azurearmtools.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-azurearmtools) [![Release Status](https://img.shields.io/github/release/Microsoft/vscode-azurearmtools.svg)](https://github.com/Microsoft/vscode-azurearmtools/releases)
```

Comment:
nit: The other extensions have the badges on the next line. I think that looks better